### PR TITLE
fix(add): add handling for topScope

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**

BugFix For #795

**Did you add tests for your changes?**
No
**If relevant, did you update the documentation?**
No
**Summary**

#795
**Does this PR introduce a breaking change?**

No
**Other information**

### CODE REVIEW

File: packages/generators/add-generator.ts [hunk 1]
```diff
@@ -161,7 +161,17 @@ export default class AddGenerator extends Generator {
 							this.configuration.config.webpackOptions.entry = entryOptions;
 							this.configuration.config.item = action;
 						});
-				}
+					} else {
+						if (action === "topScope") {
+							return this.prompt([
+								Input("topScope", "Enter line you want to add to topScope"),
```
Comment (misterdev) [inline packages/generators/add-generator.ts]: What about: "What do you want to add to topScope?"

**Comment by misterdev** [comment]
Hi @rishabh3112, great job, I confirm the error is gone.

I've copied `packages/generators/add-generator.ts` in my local fork and build it. When I run `add` on Windows it adds the line 2 times to `webpack.config.js`:

bugrishab
 
Does it happens to you? If not, I'll try again after cleaning my folders 👍

File: packages/generators/add-generator.ts (hunk 1 above)
Comment (evenstensberg) [reply packages/generators/add-generator.ts]: I agree on above

**Comment by rishabh3112** [comment]
@misterdev Thanks for reporting :100:  
This came to be another error with how transformations are handled internally. Fixed it and it worked for me now. You can check again.

Have rephrased question @evenstensberg & @misterdev :+1:

**Comment by misterdev** [comment]
Works now, well done 👍

**Comment by rishabh3112** [comment]
Hi @sendilkumarn, Can you guide me here? Currently, we are not testing any of the generators ( also smoke tests for scaffold functions). I would love to add them!

**Comment by evenstensberg** [comment]
We don't really have a good way to test scaffolds @sendilkumarn

**Pull request merged**